### PR TITLE
Move a little more tests between the minitest groups on circleci

### PR DESCRIPTION
**Pull request by coolo**
master: group 1: 4:25 minutes, group 2: 22:09 minutes
this PR: group1: 12:36 minutes, group 2: 13:54 minutes

-> 8 minutes win

### CODE REVIEW

**Comment by hennevogel** [comment]
Something is fishy here. The reported CCI status is from another pull request?

**Comment by coolo** [comment]
@bgeuken did some mean tests with circleci - and the reported status seems to be bound to the commit not to the PR

**Comment by coolo** [comment]
I can rebase to sort this out - but that would just waste time. #5372 was just a clone and the tests passed on both :)

File: src/api/lib/tasks/test_webui.rake [hunk 1]
```diff
@@ -11,22 +11,46 @@ Rake::TestTask.new do |t|
   t.warning = false
 end
 
+# minitests are a little fragile when it comes to run out of order
+# so we cherry pick some functional tests into group1 that are safe
```
Comment (hennevogel) [inline src/api/lib/tasks/test_webui.rake]: How exactly did you determine them to be safe?

Comment (coolo) [reply src/api/lib/tasks/test_webui.rake]: safe here means they don't require previous jobs and don't interfere with jobs after that. build, source, publish and maintenance tests are a nightmare to parallize

Comment (hennevogel) [reply src/api/lib/tasks/test_webui.rake]: > safe here means they don't require previous jobs and don't interfere with jobs after that. 

Please add this to the comment

Comment (coolo) [reply src/api/lib/tasks/test_webui.rake]: extended the comment

**Comment by coolo** [comment]
As codecov still doesn't comment, please note that coveralls reports a drop in coverage: https://coveralls.io/builds/18003970

**Comment by hennevogel** [comment]
> please note that coveralls reports a drop in coverage

because?

**Comment by coolo** [comment]
because the coverage of those parts is very indirect - so I call this a fair drop

**Pull request merged**